setup.py

```diff
@@ -32,7 +32,7 @@ def read(*files):
     author='UNICEF',
     author_email='rapidpro@unicef.org',
     description='',
-    # long_description=read('README.rst'),
+    long_description=read('README.rst'),
```

Comment:
Make sure this does not break the building of `sdist`, and `bdist_wheel`

Reply:
it doesn't!